apps/src/music/views/PackDialog.tsx

```diff
@@ -33,23 +40,35 @@ const PackEntry: React.FunctionComponent<PackEntryProps> = ({
   const isPlayingPreview = previewSound && playingPreview === soundPath;
   const imageSrc = library?.getPackImageUrl(folder.id);
 
+  const onEntryClick = useCallback(() => {
+    onSelect(folder);
+
+    if (usePackDialogPreview) {
+      if (soundPath && !isPlayingPreview) {
+        onPreview(soundPath);
+      }
+    }
```

Comment:
Should this be smooshed into a single conditional with another and?

```
if (usePackDialogPreview && soundPath && !isPlayingPreview) {
...
```
?

Reply:
I like keeping the `usePackDialogPreview` one separate because if the experiment is a success then we'll just take that line out when we switch to the new variant.